src/main/java/com/igot/cb/interest/service/impl/InterestServiceImpl.java

```diff
@@ -84,6 +96,7 @@ public CustomResponse createInterest(JsonNode interestDetails) {
       String interestId = String.valueOf(interestIdUuid);
       interest.setInterestId(interestId);
       ((ObjectNode) interestDetails).put(Constants.INTEREST_ID_RQST, String.valueOf(interestId));
+      ((ObjectNode) interestDetails).put(Constants.ORG_NAME, orgName);
```

Comment:
Why you do type casting on each object what is interest detail object

Reply:
InterestDetails is the jsonNode which is the payload, in the payload I need to put extra these fields to store in es so using these typecasting as directly we cannot put values to jsonNode